templates/forms/resend-key.php

```diff
@@ -54,7 +54,7 @@
 HTML;
 	}
 
-	$message_above_input_field = fs_esc_html_inline( "Enter the email address you've used for the upgrade below and we will resend you the license key.", 'ask-for-upgrade-email-address', $slug );
+	$message_above_input_field = fs_esc_html_inline( "Enter the email address you've used " . $fs->is_only_premium() ? 'during the purchase' : 'for the upgrade below' . ' and we will resend you the license key.', 'ask-for-upgrade-email-address', $slug );
```

Comment:
@xiahengchen , I am not sure whether these has been done or not, but from now on, let us **NOT** use interpolated strings inside `fs_esc_html_inline`. Those strings are supposed to be used for translations and they need to be "non interpolated".

So instead of what you've done, we need to do something like this:

```php
$message_above_input_field = $fs->is_only_premium() ?
    fs_esc_html_inline( 'full text in single quote goes here', 'ask-for-upgrade-email-address-premium-only' ) :
    fs_esc_html_inline( '...', 'ask-for-upgrade-email-address' );

```